packages/ui/src/2_molecules/VerticalTabs/components/VerticalTabItem.tsx

```diff
@@ -22,8 +23,12 @@ export const VerticalTabItem: FC<VerticalTabsItemButtonProps> = ({
       data-active={active}
       onClick={onClick}
     >
-      <p className={styles.label}>{label}</p>
-      {infoText && <small className={styles.info}>{infoText}</small>}
+      <Paragraph size={ParagraphSize.base}>{label}</Paragraph>
```

Comment:
This is probably just an outdated design, but `label` has `font-size: 1rem` in the design and `0.875rem` in the implementation.

<img width="1694" alt="image" src="https://user-images.githubusercontent.com/14878713/200604056-388a4d39-d8ca-42d4-a0a8-edf99133f94b.png">

Reply:
Since we now have the styling in VerticalTabs matching other usages of these components, we should ask Gilad to update the styling on figma to match, or use the larger text sizes here - I'll ask him to confirm

Reply:
yeah, i think it should be as Paragraph component  gives to make all paragraphs are in the same size. So current one should be correct one